src/Task/TwigCs.php

```diff
@@ -57,6 +59,12 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->addOptionalArgument('--ruleset=%s', $config['ruleset']);
         $arguments->addOptionalArgument('--ansi', true);
 
+        $arguments->addSeparatedArgumentArray('--exclude', $config['exclude']);
+
+        foreach ($config['exclude'] as $exclude) {
```

Comment:
Currently you are adding the exclude items twice. is that for demonstrating your comment or some forgotten statements?

Reply:
Sorry, it was atest.